test/core/test_serde_dataclass.py

```diff
@@ -20,14 +20,13 @@
 @serde.dataclass
 class Estimator:
     prediction_length: int
-    context_length: int = serde.OrElse(
-        lambda prediction_length: prediction_length * 2
-    )
```

Comment:
Python 3.11 would complain here, that the default value in a dataclass cannot be mutable